01-import_and_maxfilter.py

```diff
@@ -176,31 +177,44 @@ def find_bad_channels(raw, subject, session, task, run):
     tsv_data.to_csv(bads_tsv_fname, sep='\t', index=False)
 
 
-def apply_maxwell_filter(raw, subject, session, dev_head_t):
-    msg = 'Applying Maxwell filter.'
-    logger.info(gen_log_message(message=msg, step=1,
-                                subject=subject, session=session))
+def load_data(bids_basename):
+    # read_raw_bids automatically
+    # - populates bad channels using the BIDS channels.tsv
+    # - sets channels types according to BIDS channels.tsv `type` column
+    # - sets raw.annotations using the BIDS events.tsv
 
-    # Warn if no bad channels are set before Maxfilter
-    if not raw.info['bads']:
-        msg = '\nFound no bad channels. \n '
-        logger.warn(gen_log_message(message=msg, subject=subject,
-                                    step=1, session=session))
+    params = _parse_bids_filename(bids_basename, verbose=False)
```

Comment:
should this be made a public function in `mne-bids`?

Reply:
Yes maybe this would make sense!